zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -2564,7 +2564,51 @@ paths:
                 $ref: "#/components/schemas/ProblemDetail"
         "500":
           $ref: "#/components/responses/InternalServerError"
-  /mapping-rules/{mappingKey}:
+  /mapping-rules/{id}:
+    put:
+      tags:
+        - Mapping rule
+      operationId: updateMappingRule
+      summary: Update mapping rule
+      description: |
+        Update a mapping rule
```

```suggestion
        Update a mapping rule.
```

🔧 `description` should read as a sentence, as it is used as a paragraph in the docs.